docs/graphql/manual/guides/integrations/cognito-jwt.rst

```diff
@@ -0,0 +1,14 @@
+.. _cognito_jwt:
```

Comment:
The file name should be "AWS Cognito" and not "...JWT" because we are covering both the modes.